gp-includes/things/translation-set.php

```diff
@@ -260,6 +260,11 @@ public function import( $translations, $desired_status = 'current' ) {
 			 */
 			$entry->status = apply_filters( 'gp_translation_set_import_status', $is_fuzzy ? 'fuzzy' : $desired_status );
 
+			$entry->warnings = maybe_unserialize( GP::$translation_warnings->check( $entry->singular, $entry->plural, $entry->translations, $locale ) );
+			if ( ! empty( $entry->warnings ) ) {
+				$entry->status = 'waiting';
```

Comment:
That's clever but the status gets overwritten by `$entry->status = $is_fuzzy ? 'fuzzy' : $desired_status;` (line 311).

I'm actually not sure if the line is still needed since it's already set in line 261.